Dockerfile

```diff
@@ -1,26 +1,14 @@
 # Build the manager binary
-FROM golang:1.12.5 as builder
+FROM alpine
 
-WORKDIR /workspace
-# Copy the Go Modules manifests
-COPY go.mod go.mod
-COPY go.sum go.sum
-# cache deps before building and copying source so that we don't need to re-download as much
-# and so that source changes don't invalidate our downloaded layer
-RUN go mod download
 
-# Copy the go source
-COPY main.go main.go
-COPY api/ api/
-COPY controllers/ controllers/
-COPY internal/ internal/
 
-# Build
-RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 GO111MODULE=on go build -a -o manager main.go
+RUN apk add -U --no-cache ca-certificates
 
-# Use distroless as minimal base image to package the manager binary
-# Refer to https://github.com/GoogleContainerTools/distroless for more details
-FROM gcr.io/distroless/static:latest
-WORKDIR /
-COPY --from=builder /workspace/manager .
-ENTRYPOINT ["/manager"]
+FROM scratch
```

Comment:
Check if we need the distroless image from gcr or if scratch is enough!

Reply:
scratch is enough :)